MessageBird/ConversationsClient.cs

```diff
@@ -0,0 +1,129 @@
+using MessageBird.Net;
+using MessageBird.Objects.Conversations;
+using MessageBird.Resources.Conversations;
+using MessageBird.Utilities;
+
+namespace MessageBird
+{
+    public partial class Client
+    {
+        public ConversationList ListConversations(int limit = 20, int offset = 0)
+        {
+            var resource = new ConversationLists();
+
+            var list = (ConversationList) resource.Object;
+            list.Limit = limit;
+            list.Offset = offset;
+
+            restClient.Retrieve(resource);
+
+            return resource.Object as ConversationList;
+        }
+
+        public Conversation ViewConversation(string id)
+        {
+            ParameterValidator.IsNotNullOrWhiteSpace(id, "id");
+            var resource = new Conversations(new Conversation {Id = id});
```

Comment:
Perhaps it's nice to have a blank line after the parameter validation? (Applies to nearly all methods)